### PR TITLE
Improve sub-data

### PR DESCRIPTION
**Pull request by conconnl**
Improve and translation Page Titles, Meta Information

### CODE REVIEW

**Comment by conconnl** [comment]
@brianteeman I added all "Page Titles", "Page Headings" and "Meta Description" currently used on the Joomla Downloads website.
I'm planning to ask for translations too get the various pages more localized.
Could you maybe check the text?

**Comment by brianteeman** [comment]
Official languages or accredited languages? Needs to be consistent.

In almost every case I would delete the word CMS. It's not part of the name

**Comment by conconnl** [comment]
That is why I asked your help.... 😄 
I copied this out of the website, don't know who wrote it.

I would think "Official Languages", because Official is also used for the other things.
Then again the people making the packages are authorized to do it, so "Accredited"
What is in your vision the best thing too use?

File: language/en-GB/en-GB.site_menuitems.ini [hunk 1]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
 MENU_DOWNLOADS_ITEM6="Technical Requirements"
 MENU_DOWNLOADS_ITEM7="FAQs"
 MENU_DOWNLOADS_ITEM8="API Documentation"
+
+; Menus: Page Titles, Headings and Meta Descriptions
+MENU_DOWNLOADS_HOME_PAGETITLE="Build your website with the CMS Joomla!"
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:20]: `MENU_DOWNLOADS_HOME_PAGETITLE="Build your website with Joomla!"`

File: language/en-GB/en-GB.site_menuitems.ini [hunk 2]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
...
+MENU_DOWNLOADS_HOME_METADESC="Download the free open source CMS Joomla! and start building your awesome website."
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:21]: MENU_DOWNLOADS_HOME_METADESC="Download Joomla, the free Open Source CMS and start building your awesome website."

File: language/en-GB/en-GB.site_menuitems.ini [hunk 3]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
...
+MENU_DOWNLOADS_LATEST_PAGETITLE="Joomla! Downloads - Download the latest version of the CMS"
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:22]: `MENU_DOWNLOADS_LATEST_PAGETITLE="Joomla! Downloads - Download the latest version!"`

File: language/en-GB/en-GB.site_menuitems.ini [hunk 4]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
...
+MENU_DOWNLOADS_LATEST_PAGEHEADING="Latest Joomla! Release"
+MENU_DOWNLOADS_LATEST_METADESCRIPTION="Download the latest release of Joomla. Update your site or build a new one."
+MENU_DOWNLOADS_DOWNLOADS_PAGETITLE="Joomla! Downloads - All releases of the open source CMS Joomla!"
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:25]: `MENU_DOWNLOADS_DOWNLOADS_PAGETITLE="Joomla! Downloads - All releases of the Open Source Joomla!"`

File: language/en-GB/en-GB.site_menuitems.ini [hunk 5]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
...
+MENU_DOWNLOADS_LPACKS_J3_PAGETITLE="Joomla 3.x Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J25_PAGETITLE="Joomla 2.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J15_PAGETITLE="Joomla 1.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J10_PAGETITLE="Joomla 1.0 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_TECHREQ_PAGETITLE="Technical Requirements to run your Joomla! Website"
+MENU_DOWNLOADS_TECHREQ_PAGEHEADING="Technical Requirements"
+MENU_DOWNLOADS_TECHREQ_METADESC="Discover what are the technical requirements to run a website using the CMS Joomla, depending on its release."
+MENU_DOWNLOADS_FAQS_PAGETITLE="Documentation & Tutorials to build your website"
+MENU_DOWNLOADS_FAQS_PAGEHEADING="Joomla! Downloads Documentation & Frequently Asked Questions"
+MENU_DOWNLOADS_FAQS_METADESC="You have questions about how to build a powerful website using the CMS Joomla? Browse our Downloads FAQs!"
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:43]: `MENU_DOWNLOADS_FAQS_METADESC="You have questions about how to build a powerful website using Joomla? Browse our Downloads FAQs!"`

File: language/en-GB/en-GB.site_menuitems.ini [hunk 6]
```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
...
+MENU_DOWNLOADS_LPACKS_PAGETITLE="Official Joomla! Language Packages"
+MENU_DOWNLOADS_LPACKS_METADESC="Browse and download the Joomla! Official Language Packages"
+MENU_DOWNLOADS_LPACKS_J4_PAGETITLE="Joomla 4 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J3_PAGETITLE="Joomla 3.x Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J25_PAGETITLE="Joomla 2.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J15_PAGETITLE="Joomla 1.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J10_PAGETITLE="Joomla 1.0 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_TECHREQ_PAGETITLE="Technical Requirements to run your Joomla! Website"
+MENU_DOWNLOADS_TECHREQ_PAGEHEADING="Technical Requirements"
+MENU_DOWNLOADS_TECHREQ_METADESC="Discover what are the technical requirements to run a website using the CMS Joomla, depending on its release."
```
Comment (conconnl) [inline language/en-GB/en-GB.site_menuitems.ini:40]: `MENU_DOWNLOADS_TECHREQ_METADESC="Discover the technical requirements to run a website using the Joomla, depending on its release."`

**Comment by conconnl** [comment]
@brianteeman kind reminder.
Could you reply on my message and share your opinion and maybe check my suggestions as well, before we process them?

**Comment by brianteeman** [comment]
@conconnl kind response https://github.com/joomla/joomla-websites/issues/1659#issuecomment-907769328

**Pull request merged**